### DISCUSSION TITLE
为什么不适用webpack呢

### DISCUSSION THREAD

**Discussion by 1923998238** [ci010/electron-vue-next#51] [General]
为什么不适用webpack而使用rollup呢？

**Comment by ci010** [comment]
因为 vite 是基于 rollup 的，最后 build 也是用 rollup build 的。

当然因为 electron 分成 main 和 renderer，你如果想用 webpack 来 build main 也可以（renderer 依旧是 rollup，因为 vite 设计如此）。只不过加上 webpack 之后，你的 dependencies 会更乱……因为你现在有2个独立的 build 系统……

不过如果很多人觉得 webpack 更好我可以加上 webpack + vue 2/3 的组合，毕竟我之前的项目就是这个配置
